tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/utils/git.ts

```diff
@@ -1,9 +1,45 @@
 import * as child_process from "child_process";
-
+import { execSync } from "child_process";
+import simpleGit, { SimpleGit } from 'simple-git';
 
 export function getFileListInPackageFolder(packageFolder: string) {
-    child_process.execSync('git add .',{ encoding: "utf8", cwd: packageFolder });
-    const files = child_process.execSync('git ls-files',{ encoding: "utf8", cwd: packageFolder }).trim().split('\n');
+    child_process.execSync('git add .', {encoding: "utf8", cwd: packageFolder});
+    const files = child_process.execSync('git ls-files', {encoding: "utf8", cwd: packageFolder}).trim().split('\n');
 
     return files;
 }
+
+export function getHeadSha(specRepo: string) {
+    const headSha = execSync(`git rev-parse HEAD`, {encoding: "utf8", cwd: specRepo});
```

Comment:
I found you already import the simple git library, could you use that API of that lib directly instead of combining git commands.

Reply:
I don't find the simple git library has this functionality. So I only can use git command.

Reply:
I also see other parts like git add etc, do you mean none of them is supported in simple git?

Reply:
No, most functionality can be supported by simple-git, but some functionality is not supported, such as `--replace-all` flag in git config. By the way, I will remove the `simple-git` to use command directly in all places to keep consistent.